### DISCUSSION TITLE
Smokeview Issue with Vector's Length

### DISCUSSION THREAD

**Discussion by CliveFirst1234** [firemodels/fds#12877] [Q&A]
Hello, I have a current issue with vector visualization on Smokeview.
I have a pipe, with airflow in it, and a hole appears in the middle of the pipe. I need to check if air comes out of the pipe near the hole, and for that I want to visualize the speed vectors to check if some of them come out of the pipe or not. I have a problem when doing the visualization on Smokeview. There is too large vectors around the hole, and I can't get a correct view of the results (I'm doing a millimetric mesh around the are of interest, because of the size of the pipe).
When checking the SMV User's guide, I only saw the keybind "a" ("The vector lengths may be adjusted by depressing the “a” key"), but when I use it it only extends the length of vectors. Also I can't use the keybind "s" (The “s” key may be depressed to alter the
number of vectors displayed") because It may suppress the vectors I want to see.
So my question is, is there a way to reduce the length of vectors on Smokeview ?
Thank you in advance for your answer, and I hope this wasn't answer before.

**Comment by gforney** [comment]
Press ALT a to reduce vector length (not A). Also, vector length can be controlled using the slice
settings dialogue box

**Reply by CliveFirst1234** [reply]
Thank you very much !

**Comment by gforney** [comment]
I edited my comment after first posting.  make sure you use ALT a to decrease.  I'll close the issue. reopen if you see a problem
